### PR TITLE
fixed the feather C

### LINKED ISSUE #4: Feature C to a task
fix the settings, to add the static file route.
settings.py will be changed.

### PR DESCRIPTION
**Pull request by huih1026**
i have finished the featherC, please review and merge

### CODE REVIEW

File: mysite/mysite/settings.py [hunk 1]
```diff
@@ -118,6 +118,7 @@
 # https://docs.djangoproject.com/en/3.2/howto/static-files/
 
 STATIC_URL = '/static/'
+STATIC_ROOT = BASE_DIR / "staticfiles"
```
Comment (kinming92) [inline mysite/mysite/settings.py:122]: Please add a new line

File: mysite/mysite/settings.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['localhost']
```
Comment (kinming92) [inline mysite/mysite/settings.py:29]: add another line

File: mysite/mysite/settings.py (hunk 1 above)
Comment (huih1026) [reply mysite/mysite/settings.py:122]: okay

**Review by kinming92 — CHANGES_REQUESTED** [review]
change debug to false

**Review by kinming92 — APPROVED** [review]
Look good to me.

**Pull request merged**